config/202-gateway.yaml

```diff
@@ -37,13 +22,30 @@ spec:
     tls:
       mode: PASSTHROUGH
 ---
-# TODO(#1969): We should allow the users to use `istio-ingressgateway` by default,
-#              while having the choice to specify their own Ingress Gateway service
-#              if that isn't enough for them.  That way we can get out of maintaining
-#              these YAML ourselves.
+# We stand up a new Gateway service to receive all external traffic
+# for Knative pods.  These pods are basically standalone Envoy proxy
```

Comment:
`These pods` here is a little bit confusing. I think they refer to the pods that the Gateway service is backed by, right? We may want to directly call this out.